### ISSUE TITLE
Bug: stuck on loading when offline

### ISSUE THREAD

**Issue by devilkkw** [Nevysha/Cozy-Nest#37]
If you launch sd offline, the ui isn't loaded, stuck ad launching screen.
No error in console, sd is loaded correctly, but UI don't start.
If you are online,all work.

**Comment by Nevysha** [comment]
Yes, that's because of external library I'm getting through CDN. I'll add them to the extension in the next update

**Comment by ostap667inbox** [comment]
I'm online, but still SD hangs on loading, unfortunately I have to delete the extension, it does not allow to work.
![chrome_ytKMu99DOo](https://github.com/Nevysha/Cozy-Nest/assets/123593542/286ee8d5-fc28-4734-bde0-1acb930fa5e8)

**Comment by Nevysha** [comment]
As you want. Please note that you can add `CozyNest=No` to URL to disable the extension. 

If you still want to try out to fix that, please provide me with your extension list, commit hash of a1111 and browser. (And please open dedicated Issue if possible)

**Comment by ostap667inbox** [comment]
Browser: Chrome 113.0.5672.93
A1111 commit hash: [5ab7f21](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/5ab7f213bec2f816f9c5644becb32eb72c8ffb89)
List of extensions (all updated):
![chrome_rTMAlIMWph](https://github.com/Nevysha/Cozy-Nest/assets/123593542/bd97f057-f365-476e-8c26-5d699ff271f5)

**Comment by Nevysha** [comment]
Yes I'll work to fix this.

**Comment by Nevysha** [comment]
It should be fixed if you update Cozy Nest now

**Comment by ostap667inbox** [comment]
> It should be fixed if you update Cozy Nest now

Great, now everything is fine. Thank you very much!
